File: src/main/java/com/giant/auth/dto/request/SignInRequestDto.java
package com.giant.auth.dto.request;

import jakarta.validation.constraints.NotBlank;

public record SignInRequestDto(
        @NotBlank(message = "아이디를 입력해주세요.")
        String userName,
        @NotBlank(message = "비밀번호를 입력해주세요.")
        String password,
        boolean isAuto
) {
}

File: src/main/kotlin/com/giant/module/auth/dto/request/SignInRequestDto.kt
package com.giant.module.auth.dto.request

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotBlank

@Schema(name = "SignInRequest")
data class SignInRequestDto(
    @field:NotBlank(message = "아이디를 입력해주세요.")
    @get:Schema(description = "아이디", example = "exampleid1234")
    val userName: String,


    @field:NotBlank(message = "비밀번호를 입력해주세요.")
    @get:Schema(description = "비빌번호", example = "examplepw1234!@")
    val password: String,

    @get:Schema(description = "장기 로그인", example = "false")
    val isAuto: Boolean = false
)